test/ServiceStack.EventStore.IntegrationTests/TestAppHost.cs

```diff
@@ -21,18 +22,18 @@ public class TestAppHost : AppHostHttpListenerBase
 
         public override void Configure(Container container)
         {
-            var settings = new SubscriptionSettings()
-                .SubscribeToStreams(streams =>
-                {
-                    streams.Add(new ReadModelSubscription()
-                        .SetRetryPolicy(1.Seconds(), 3.Seconds())
-                        .WithStorage(new ReadModelStorage(StorageType.Redis, "localhost:6379")));
-                });
 
             LogManager.LogFactory = new ConsoleLogFactory();
 
+            var nodeBuilder = EmbeddedVNodeBuilder.AsSingleNode()
+                           .OnDefaultEndpoints()
+                           .RunInMemory();
+            var node = nodeBuilder.Build();
+            node.StartAndWaitUntilReady().Wait();
```

Comment:
I took the code for building the embedded node from the EventStore documentation which was written after the StackOverflow post. What I will do is write some tests to replicate the problem in the SO post.